### ISSUE TITLE
Typescript `Duplicate identifier`error after 3.6.1 mobx update

### ISSUE THREAD

**Issue by mturki** [mobxjs/mobx#1362]
I have a **Duplicate identifier 'require'** error with my Typescript react native project when i run the `tsc` command :
`node_modules/@types/node/index.d.ts(99,13): error TS2300: Duplicate identifier 'require'.
node_modules/@types/react-native/index.d.ts(8618,14): error TS2300: Duplicate identifier 'require'.`

Downgrading to mobx 3.5.1 solves the problem.

typescript version : 2.7.2

**Comment by mweststrate** [comment]
The error doesn't mention anything related to mobx, but two other packages instead (and mobx doesn't depend on either of them). So I suspect this error originates from another update that occurred at the same time, or a mess in your node_modules.

Please reopen and provide a reproduction repo if you think this isn't the case and the issue is actually caused by MobX.

**Comment by mturki** [comment]
I tried to clear node_modules without effect. Only a downgrade to 3.51 fix my issue. I create a minimal project from scratch to reproduce it. And now i have a more explicit mobx related error when i run `tsc` : 
`node_modules/mobx/lib/utils/utils.d.ts(1,23): error TS2688: Cannot find type definition file for 'node'.`
Any idea?

I posted my minimal code there : [https://github.com/mturki/testMobx](https://github.com/mturki/testMobx)
